webapp/channels/src/components/create_team/components/team_url/team_url.test.tsx

```diff
@@ -65,15 +50,28 @@ describe('/components/create_team/components/display_name', () => {
         const actions = {...defaultProps.actions, checkIfTeamExists};
         const props = {...defaultProps, actions};
 
-        const wrapper = mountWithIntl(
+        renderWithContext(
             <TeamUrl {...props}/>,
         );
 
-        await (wrapper.instance() as unknown as TeamUrl).submitNext({preventDefault: jest.fn()} as unknown as React.MouseEvent<Button, MouseEvent>);
+        screen.getByText('Finish').click();
+
+        // Wait for the check to see if the team already exists which should fail
+        await Promise.resolve();
+
+        expect(screen.getByText('This URL is taken or unavailable. Please try another.')).toBeInTheDocument();
+
         expect(actions.checkIfTeamExists).toHaveBeenCalledTimes(1);
         expect(actions.createTeam).not.toHaveBeenCalled();
 
-        await (wrapper.instance() as unknown as TeamUrl).submitNext({preventDefault: jest.fn()} as unknown as React.MouseEvent<Button, MouseEvent>);
+        expect(screen.getByText('This URL is taken or unavailable. Please try another.')).toBeInTheDocument();
```

Comment:
Nothing should have changed between the first `expect(screen...)` and the second, right? There is no need to check twice.

Reply:
Good catch. I'll remove this